packages/odyssey-react/src/components/Infobox/Infobox.theme.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  // Root
+  FontLineLengthMax: theme.FontLineLengthMax,
+  SpacePadding: theme.SpaceRemM,
+  SpaceMarginBlockEnd: theme.SpaceRemM,
+  BorderRadius: theme.BorderRadiusBase,
+  ColumnGap: theme.SpaceRemS,
+
+  IconSize: theme.FontSizeHeading5,
+  IconLineHeight: theme.FontLineHeightTitle,
+
+  ContentSpaceMarginBlockEnd: theme.SpaceRemXs,
```

Comment:
@jeffbelser-okta If this format is starting to feel verbose, maybe we can drop type names like `Space` for local variables. In this case, noting `Margin` implies the type of value.

I'm not fretting about verbosity yet, but wanted to offer an alternative if we aim to be more terse.

Reply:
Stylesheets that stress the limits of our naming scheme like this are a code smell. We should consider a new Icon or Content component stylesheet that can keep its variables separate. More like Table perhaps. It does mean there are more `[Component.theme]` entries to make at run time but we can document that and work toward better type support.